resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/delegates/ClientCookieHeaderDelegate.java

```diff
@@ -0,0 +1,29 @@
+package org.jboss.resteasy.plugins.delegates;
+
+import javax.ws.rs.core.NewCookie;
+
+import org.jboss.resteasy.resteasy_jaxrs.i18n.Messages;
+
+/**
+ * 
+ * @author <a href="mailto:ron.sigal@jboss.com">Ron Sigal</a>
+ * @date May 20, 2016
+ */
+public class ClientCookieHeaderDelegate extends NewCookieHeaderDelegate
+{
+   public String toString(Object value)
+   {
+      if (value == null)
+      {
+         throw new IllegalArgumentException(Messages.MESSAGES.paramNull());
+      }
+      NewCookie cookie = (NewCookie) value;
+      StringBuilder b = new StringBuilder();
+      b.append(cookie.getName()).append('=');
+      if (cookie.getValue() != null)
+      {
+         quote(b, cookie.getValue());
```

Comment:
Sorry, I actually meant RFC 2109, not RFC 2965 (which tried to replace everything with new headers, and obviously failed; I think Opera is the only browser that ever implemented it, and it no longer does now that it switched to Chromium); but it doesn't change the issue: RFC 2109 is not used in practice; everyone follows "the original Netscape spec", adding extensions over time (Secure, HttpOnly), that were never spec'd until RFC 6265 aimed at describing the current state of the web (similar to how HTML5 also describes the current state of browser parsing behavior, besides adding new features).

I have no problem with breaking JAX-RS-spec compliance by dropping RFC 2109, but I'm almost sure there are some TCK tests that would require it, and there may be some "Enterprisey" people relying on it (there must be reasons JRE `java.net.HttpCookie`, Servlets and JAX-RS support, or are based on, those RFCs).

BTW, FYI, the JAX-RS spec bug report: https://java.net/jira/browse/JAX_RS_SPEC-430